src/jsMain/kotlin/io/beatmaps/user/alerts/alert.kt

```diff
@@ -78,11 +80,13 @@ class AlertElement : RComponent<AlertProps, AlertState>() {
                 if (props.hidden == true) {
                     height = "0px"
                     opacity = "0"
+                    margin = "-1px 5px" // -1 pixel to account for the border
```

Comment:
If you're trying to account for the border.... why not change the border?

Reply:
As far as I could tell, the border width couldn't be animated with `transition`.

Reply:
`transition: opacity .2s, height .2s linear .2s, border-width .2s linear .2s`
works just fine if you actually define that property